evm-contracts/src/v0.6/dev/AggregatorProxy.sol

```diff
@@ -11,9 +12,16 @@ import "./Owned.sol";
  * trusted to update it.
  */
 contract AggregatorProxy is AggregatorInterface, AggregatorV3Interface, Owned {
+  using SafeMath for uint256;
 
+  uint16 public epoch;
```

Comment:
We can save 800 gas on `latestRoundData` by packing epoch and aggregator into a struct and storing that struct instead.